rules/migrate-an-existing-user-store-to-an-externalauthprovider/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 title: Do you know how to migrate an existing user store to an ExternalAuthProvider?
-uri: integrate-identityserver-with-an-existing-user-store
+uri: migrate-an-existing-user-store-to-an-externalauthprovider
```

Comment:
1. Can you also check if any categories list the old uri. 
2. Please also update the uri if you find any